packages/api/src/shared/numbers.ts

```diff
@@ -0,0 +1,8 @@
+export function getFloatValue(value: string | number): number {
+  return typeof value === "string" ? parseFloat(value) : value;
+}
+
+// Truncate the number to 2 decimal places
+export function formatNumber(num: number): number {
+  return parseInt((num * 100).toFixed(2)) / 100;
```

Comment:
NABD, but why `.toFixed(2)`?

This is a good candidate for some unit testing! 🤓

Reply:
2 decimal places seems like a reasonable cutoff for most measurements. Dima and I had this conversation a while ago when I was fixing a bug on weight measurements with Fitbit

Reply:
That's obtained by multiplying by 100 and converting to int. I don't see the reason for the `.toFixed(2)`. Did some [tests on JSFiddle](https://jsfiddle.net/2eya4ws3/5/), which confirms this.

NABD, but if it can be simplified its better for understanding.

Also, we have other places doing this, maybe we can refer to this new function?

Reply:
Oh, yea, that makes sense

Reply:
Fixed